### PR TITLE
Adaptive pager

### PR DESCRIPTION
**Pull request by magdalenaan**
closes: https://github.com/telerik/kendo-themes-private/issues/161

### CODE REVIEW

File: packages/html/src/grid/tests/grid-adaptive-pager.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export default () =>(
                         <div className="k-view k-widget">
                             <div className="k-stretched-view k-content">
                                 <div className="k-grid k-grid-md k-grid-display-block k-grid-mobile k-editable">
-                                    <Pager className="k-grid-pager" mobile="small" />
+                                    <Pager className="k-grid-pager" adaptive info={false} pageSizes={false} />
```
Comment (Juveniel) [inline packages/html/src/grid/tests/grid-adaptive-pager.tsx:15]: are info and pageSize always hidden in adaptive mode ?

Comment (Juveniel) [reply packages/html/src/grid/tests/grid-adaptive-pager.tsx:15]: i suggest to leave it as is for now

File: tests/_output/default/pager/pager-adaptive.png
Comment (inikolova) [inline tests/_output/default/pager/pager-adaptive.png:1]: According to the design, there shouldn't be any margins around the page dropdown/select element and the Prev/Next buttons in the smallest size

Comment (inikolova) [reply tests/_output/default/pager/pager-adaptive.png:1]: After offline sync with the designers, there is a decision that the PageSizes dropdown won't be visible; also there will be margins around the page dropdown/select element and the Prev/Next buttons. The design has been updated https://www.figma.com/file/VCIOKDxHOw7qYkC4ks5Ctp/Adaptive-Story?type=design&node-id=2015-210910&mode=design&t=EhnykUsopnUJ2wVB-0

**Review by inikolova — APPROVED** [review]

**Review by Juveniel — APPROVED** [review]

**Pull request merged**